src/xdocs/config_blocks.xml

```diff
@@ -395,6 +395,9 @@ switch (a) {
       </subsection>
 
       <subsection name="Examples" id="EmptyCatchBlock_Examples">
+        <p>
+            To configure the check:
+        </p>
         <p>
           To configure the check to suppress empty catch block if exception's variable name is
```

Comment:
can we update 
https://github.com/checkstyle/metadata-gen/blob/454e076e796b1af4c9d2f524a11cb9c97e56d4fd/src/main/java/org/example/JavadocMetadataScraper.java#L27

to match 
```
<p>
          To configure the check to supp
```

Two times: `To configure the check` in text does not make any sense.

You can have few markers for examples to start.

Reply:
Yes, can be achieved by the modified regex